wiki/FAQ/en.md

```diff
@@ -36,7 +36,7 @@ needs_cleanup: true
 [Taiko no Tatsujin wikipedia]: https://en.wikipedia.org/wiki/Taiko_no_Tatsujin "Wikipedia entry for Taiko no Tatsujin"
 [Skin Compendium link]: https://osu.ppy.sh/community/forums/topics/686664/ "Completed Skins Compendium by RockRoller"
 [Beatmaplist link]: https://osu.ppy.sh/p/beatmaplist "Official beatmaplist"
-[osu!tutorial basic]: https://osu.ppy.sh/beatmapsets/3756 "Installer's osu!tutorial by peppy"
+[new beginnings]: https://osu.ppy.sh/beatmapsets/1011011 "new beginnings by nekodex"
```

Comment:
seems like it's meant to be a map creator, not the song creator, ``by pishifat``? or possibly by pishifat and Noffy (she made storyboard)